workflow/exp/exp.ens_conus12km

```diff
@@ -4,84 +4,73 @@
 export NET=rrfs                    # rrfs, rtma, 
 export MESH_NAME=conus12km         # conus12km, conus3km, atl3km, etc
 export WGF="enkf"   # working group function = "det", "enkf", "ensf", "firewx"
-export EXP_NAME=rrfs${WGF}
-export VERSION=v2.0.3
-export TAG=c3v203
-export OPSROOT=/lfs5/BMC/nrtrr/NCO_dirs/${VERSION}
-export EXPDIR=${OPSROOT}/${EXP_NAME}
+export EXP_NAME=hrly_12km
+export VERSION=v2.0.8
+export TAG=c3v208
+export OPSROOT=/scratch1/BMC/wrfruc/gge/OPSROOT
+export EXPDIR=${OPSROOT}/exp/${EXP_NAME}/rrfs${WGF}
 export COMROOT=${OPSROOT}/com      # task input and output data as well as logs
 export DATAROOT=${OPSROOT}/stmp    # task workdirs ($DATA) which to be removed immediately upon task completion unless KEEPDATA=yes
 #
-export REALTIME=true     # true, false
-#
 export DO_ENSEMBLE=true
 export DO_DETERMINISTIC=false
 export ENS_SIZE=30
 #
-### preempt some settings for conus3km realtime runs on jet
 export PHYSICS_SUITE="convection_permitting"   # mesoscale_reference, convection_permitting
 export ZETA_LEVELS="L60.txt"
-export COMINgfs="/lfs5/BMC/wrfruc/Chunhua.Zhou/w4/data/GFS/from_aws"
-export COMINrap="/lfs5/BMC/nrtrr/NCO_data/rap"
-export COMINgefs="/lfs5/BMC/wrfruc/Chunhua.Zhou/w4/data/GEFS"
 #
-export FCST_ONLY="false"
+export DA_IODA="true"
+export DA_JEDI="true"
```

Comment:
Do we have this ready for ensemble? If not, we should wait until we have EnsDA ready.

Reply:
I am working on making it ready. This is the first step of a few PRs.

For those who want to continue trying the sandbox retros, we have made a tag for that purpose.
https://github.com/NOAA-EMC/rrfs-workflow/issues/609

Reply:
I got what you mean now.
To clarify,  `exp.ens_conus12km` will save for retro runs.
`exp.rt_jet_ens_conus12km` will be used for jet realtime runs. I saw your comments on it and I agree with it.  It will be great if you can double-check `exp.rt_jet_ens_conus12km` to make sure it works for jet realtime runs. Thanks!